src/components/ec-inline-input-field/components/read-only/read-only.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div
+    class="tw-w-full tw-flex tw-justify-between tw-items-center"
+    data-test="ec-inline-input-field-read-only"
+  >
+    <span class="tw-truncate">{{ value }}</span>
+    <button
+      ref="editButton"
+      class="ec-inline-input-field-read-only__action"
+      data-test="ec-inline-input-field-read-only__action"
+      @click="edit"
+      @keydown.enter.space.prevent="edit"
+    >
+      <ec-icon
+        class="ec-inline-input-field-read-only__edit-icon"
+        name="simple-edit"
+        :size="16"
+        type="interactive"
+      />
+    </button>
+  </div>
+</template>
+
+<script>
+import EcIcon from '@/components/ec-icon';
+
+export default {
+  name: 'EcInlineInputFieldReadOnly',
```

Comment:
I don't think it's the best naming, actually I think it is a bit misleading the fact that it is editable it makes it not read only (another thing is the component state).

I'd use something like `value-text.vue` or similar, not totally sure.